src/sql-editor/standardSql/standardSuggestionsRegistry.ts

```diff
@@ -57,6 +64,24 @@ export const initStandardSuggestions =
             },
           ]),
       },
+      {
+        id: SuggestionKind.ValueMacro,
+        name: SuggestionKind.ValueMacro,
+        suggestions: (_, m) =>
+          Promise.resolve([
+            ...macros
+              .list()
+              .filter((m) => m.type === MacroType.Value)
+              .map((m) => ({
+                label: m.name,
+                insertText: `${"\\" + m.text}${argsString(m.args)}`,
+                insertTextRules: CompletionItemInsertTextRule.InsertAsSnippet,
+                kind: CompletionItemKind.Function,
```

Comment:
To distinguish macros from the SQL functions:
```suggestion
                kind: CompletionItemKind.Snippet,
```